### ISSUE TITLE
Upgrading of docker containers

### ISSUE THREAD

**Issue by scornflake** [johncclayton/electric#53]
We need a way to upgrade stuff, from the WEB ui.

I guess this would do a:

a) docker pull scornflake/electric-pi
b) docker restart

**Comment by scornflake** [comment]
This is probably worth doing:

http://docs.ionic.io/services/deploy/

**Comment by johncclayton** [comment]
Yes, if you actually had people that needed it.  I have a build box here – but longer term TOTALLY WANT TO.

Thanks
--
John Clayton
CTO
Skype: johncclayton
